benchmark/bench_matrix_ops.jl

```diff
@@ -0,0 +1,41 @@
+module BenchMatrixOps
+
+using BenchmarkTools
+using LinearAlgebra
+using Random
+using StaticArrays
+
+const suite = BenchmarkGroup()
+const Nmax = 20
+
+# Use same arrays across processes (at least with the same Julia version):
+const RNG = MersenneTwister(1234)
+
+# Unary operators
+for f in [det, inv, exp]
+    s1 = suite["$f"] = BenchmarkGroup()
+    for N in 1:Nmax
+        SA = @SMatrix rand(RNG, N, N)
+        A = Array(SA)
+        s2 = s1["$N"] = BenchmarkGroup()
+        s2["SMatrix"] = @benchmarkable $f($SA)
+        s2["Matrix"] = @benchmarkable $f($A)
+    end
+end
+
+# Binary operators
+for f in [*, \]
```

Comment:
`perf/benchmark_matrix_ops.jl` only benchmarks `\`:

https://github.com/JuliaArrays/StaticArrays.jl/blob/5ead6c8693104daeccbed91dcc26dc4eb679d750/perf/benchmark_matrix_ops.jl#L5

I guess it was because `*` was handled in the extensive benchmarks in `perf/benchmark*.jl`.  But I guess it's reasonable to do this here, at least until they are ported to BenchmarkTools-based benchmarks?

Reply:
Sounds good to me.